mautic/mautic
# Rector up to symfony 4.3

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR is applying automatic refactoring of deprecated code in Symfony 4.3. There was code changed all over the place but the changes are quite simple. Just replacing the deprecated class for its new counterpart.

#### Steps to test this PR:

All of the bellow steps are quite well tested by functional test and the API-Library tests, so if we have all the checks green then this should be fairly safe to merge, but I'm writing down the affected areas if anyone wants to click through it.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make a API call to fetch all assets, campaigns, campaign events, categories, marketing messages, themes, emails, forms, form submissions, devices, custom fields, contacts, segments, notes, tags, pages, points, reports, sms, stage, role, user, weboook, focus, tweet to see that it still works
3. Try to access a route that does not exist to see if the error page is still as nice as before
4. Try to go through the installation workflow
5. Import still works
